mico-core/src/main/java/io/github/ust/mico/core/resource/DeploymentResource.java

```diff
@@ -53,7 +56,7 @@
             micoApplicationJobStatus = deploymentBroker.deployApplication(shortName, version);
         } catch (MicoApplicationNotFoundException e) {
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, e.getMessage());
-        } catch (MicoServiceInterfaceNotFoundException | MicoApplicationDoesNotIncludeMicoServiceException | DeploymentRequirementsNotMetException e) {
+        } catch (MicoServiceInterfaceNotFoundException | DeploymentRequirementsNotMetException e) {
```

Comment:
Is the removed exception still used somewhere else? If not it can be removed.

Reply:
The exception is still used by two other endpoints.